### DISCUSSION TITLE
re: English interface: visual corrections and logical perception #49

### DISCUSSION THREAD

**Discussion by chrizy66** [xyzroe/XZG#62] [Ideas]
Hi there,

I have some comments on the recent changes to the English interface.
I am not sure if the course that has been taken is an improvement or just a change.
New descriptions may lead to reduced perception of the user interface.
For example:

Status -> Device -> Firmware
Is it firmware timestamp or version after all?

Status -> Zigbee -> Socket
"Socket 2 clients" what? Connected or in some other state?

General -> LED configuration -> Night mode
"time based LEDs control"
This short additional description was very clarifying.

General -> NTP
Display only, or can it be set?

Zigbee -> Serial settings -> Device operating mode
Only display, or can it be set?
What was the sensible reason for removing "Select the ..."

Tools -> Debug -> Console -> Clear
Does it clear the console or Zigbee? Because there are Zigbee buttons nearby

In my opinion, standardizing the interface by reducing the amount of information provided is not always the best choice.

Cheers - Chris

**Comment by xyzroe** [comment]
@SAOPP please take you part

**Comment by SAOPP** [comment]
Wsup?

Glad to see the comments... I've been waiting for this. I'll answer in detail a little later, there will be something to work on...

**Comment by SAOPP** [comment]
> Status -> Device -> Firmware
> Is it firmware timestamp or version after all?

What do u mean by timestamp? U are in the device section, which means that inside u can see parameters responsible for displaying some conditional system values. The ESP firmware version number has a date, does this confuse u?

> Status -> Zigbee -> Socket
> "Socket 2 clients" what? Connected or in some other state?

Again, the client cannot be in any state other than connected.

> General -> NTP
> Display only, or can it be set?

Display what? :smiley: Is this the NTP protocol, are u seeing it for the first time or have u previously only looked at it and not configured it? Here I don't quite understand what u are expecting or what scares u.

> Zigbee -> Serial settings -> Device operating mode
> Only display, or can it be set?
> What was the sensible reason for removing "Select the ..."

Brrr :smile: U are in the Zigbee section right? Looking under the section called 'Serial **settings**', u OBVIOUSLY have a drop-down list that is named 'Device operating mode' - what’s wrong? I would especially like to hear some sensible comments ova here.

> Tools -> Debug -> Console -> Clear
> Does it clear the console or Zigbee? Because there are Zigbee buttons nearby

Am... I dunno, but maybe, u expect this to clear your root partition or call to 911? :face_in_clouds: U wrote: `Tools -> Debug -> Console` - so u r in device console right? U have control elements under the console window, have u never cleared the console or is this the first time u see that this can be done?

In general, I would like to hear some specific comments from u, since your questions often have an answer, and this answer is given from the interface itself. Yes, logic should be present in the orientation of interface elements, but this is the main point, and this is what I wanted. We are not talking 'bout a Zigbee gateway operating in a housewife’s kitchen, this is a specific device, and an approach to this device is only possible if u have a basic technical knowledge base.

**Comment by chrizy66** [comment]
@SAOPP 
I can see how narrow vision can affect judgment. This interface is intended for many people from many countries with different experience. If you can't imagine what they can - don't mess with UI designs. I'm not trying to impose anything - not my choice - and I won't discuss it with you anymore.

**Comment by SAOPP** [comment]
@chrizy66 We can discuss a lot of different variations and things, but I would like to hear some compelling argument from u. If u have something to offer, then let's start there. If u have really good suggestions for improvement, I will be only too happy. :+1:

**Comment by SAOPP** [comment]
Suppose, as u mentioned in your first post, points that are not entirely obvious to u, could u offer your vision - how would u be more comfortable understanding these options?
